Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:carbon
```

Comment:
We need to control the version of node we're running.

If we're going with async await it should be 8.x, otherwise it should be 6.x